stingray/tests/test_crossspectrum.py

```diff
@@ -1005,36 +962,10 @@ def test_make_empty_crossspectrum(self):
         assert cs.n is None
         assert cs.power_err is None
 
-    def test_no_counts_warns(self):
-        newlc = copy.deepcopy(self.lc1)
-        newlc.counts[: newlc.counts.size // 2] = 0 * newlc.counts[: newlc.counts.size // 2]
-        with pytest.warns(UserWarning) as record:
-            ps = AveragedCrossspectrum(newlc, self.lc2, segment_size=0.2, legacy=True)
-
-        assert np.any(["No counts in " in r.message.args[0] for r in record])
-
     def test_no_segment_size(self):
         with pytest.raises(ValueError):
             cs = AveragedCrossspectrum(self.lc1, self.lc2)
 
-    def test_invalid_type_attribute(self):
-        with pytest.raises(ValueError):
-            cs_test = AveragedCrossspectrum(self.lc1, self.lc2, segment_size=1, legacy=True)
-            cs_test.type = "invalid_type"
-            assert AveragedCrossspectrum._make_crossspectrum(cs_test, self.lc1, self.lc2)
-
-    def test_invalid_type_attribute_with_multiple_lcs(self):
-        with pytest.warns(UserWarning) as record:
-            acs_test = AveragedCrossspectrum(
-                [self.lc1, self.lc2], [self.lc2, self.lc1], segment_size=1
-            )
-        acs_test.type = "invalid_type"
-        with pytest.raises(ValueError) as excinfo:
-            assert AveragedCrossspectrum._make_crossspectrum(
-                acs_test, [self.lc1, self.lc2], [self.lc2, self.lc1]
-            )
-        assert "Type of spectrum not recognized" in str(excinfo.value)
-
```

Comment:
Similarly with the tests above. These don't explicitly test the legacy interface, do they? Or are the relevant versions of the tests for the new code in `test_fourier.py`?

Reply:
This error was only in `_make_crossspectrum`, and it was relevant because the same method was used for both the cross spectrum and the power spectrum. But now that the two mechanisms are separated, this test can be neglected.